### PR TITLE
Add some examples.

### PR DESCRIPTION
**Pull request by buraglio**
I added some basic examples. there may be some questions about address compression where all zeroes exist, not sure if that is an issue that we should address or not.

### CODE REVIEW

File: draft-ek-srv6ops-sidspace-experiment.md [hunk 1]
```diff
@@ -86,16 +86,34 @@ Assuming 32-bit ASNs, this yields a /48 per ASN in use within an SRv6 domain, i.
 ## SRv6 SID Documentation Prefixes
 
 Using 16-bit and 32-bit ASNs reserved for documentation purposes [IANA-ASNs] yields several SRv6 SID prefixes that might be used for SRv6 documentation purposes.
-These prefixes presently include:
+These prefixes presently include ASNs in the range of 64496-64511 as defined in [RFC5398]:
+
+~~~~~~
+5f00:6449:6:6449:7::/48
```
Comment (ekline) [inline draft-ek-srv6ops-sidspace-experiment.md]: I think these need to be hex, not decimal in a hex context.

* `5f00:0:fbf0::/48`
* `5f00:0:fbff::/48`

Comment (ekline) [reply draft-ek-srv6ops-sidspace-experiment.md]: likewise for the other ranges

Comment (buraglio) [reply draft-ek-srv6ops-sidspace-experiment.md]: Oh, duh. that makes a lot more sense. I'll fix that.

**Review by ekline — APPROVED** [review]
I think there's bit of tweaking might yet do, but this great, thank you.

**Comment by buraglio** [comment]
I agree. I am planning to think on this a bit more and maybe add some suggestions once I am back from summer holiday.

**Pull request merged**